lib/components/narrative/connected-trip-details.js

```diff
@@ -13,6 +13,8 @@ const TripDetails = styled(TripDetailsBase)`
 
 const mapStateToProps = (state, ownProps) => {
   return {
+    defaultFare: state.otp.config.itinerary?.defaultFare || undefined,
```

Comment:
Rename to `defaultFareKey` per https://github.com/opentripplanner/otp-ui/blob/master/packages/trip-details/src/index.js#L204

Reply:
Updated in a32088189205a527c35064378e6ef614edd5b98e